Document CSS bundles updates (#11526)
mozilla/bedrock: docs/coding.rst

    // bedrock/bedrock/firefox/templates/firefox/{specific-page}.html

    {% from "macros-protocol.html" import billboard with context %}

    {{ billboard(
        title='This is Firefox.',
        ga_title='This is Firefox',
        desc='Firefox is an awesome web browser.',
        link_cta='Click here to install',
        link_url='mozilla.org/firefox',
      )}}

> Because component styles are not global, we still have to import the page-specific Protocol styles in SCSS:

Reviewer: Nit: I think it reads better as "Because not all component styles are global" (there's probably a name for this rule in English grammar but I don't know it)

    $brand-theme: 'firefox';

    @import '~@mozilla-protocol/core/protocol/css/includes/lib';
    @import '~@mozilla-protocol/core/protocol/css/components/billboard';

Import CSS Bundles
~~~~~~~~~~~~~~~~~~

> We created pre-built CSS bundles to be used for said components due to their frequency of use. This method only requires an import into the HTML page. Since it’s already a CSS bundle, we don’t need to import it in the respective CSS page.

Reviewer: ```suggestion
We created pre-built CSS bundles to be used for some components due to their frequency of use. This method only requires an import into the HTML template. Since it’s a separate CSS bundle, we don’t need to import that component in the respective page CSS.
```

The CSS bundle import only works for the following components:

* Split
* Card
* Picto
* Callout
* Article
* Newsletter form
* Emphasis box

To import a respective bundle, all that is required to place the CSS bundle into the ``page_css`` block like the following:

.. code-block:: html

    {% block page_css %}
        {{ css_bundle('protocol-split') }}
        {{ css_bundle('protocol-card') }}

>     {% endblock %}

Reviewer: Should we add an example of a page bundle here as well, since most pages will have one? Like `{{ css_bundle('my-example-page) }}`

Though if we add that we should also call it out above, something like "Include a CSS bundle in the template's ``page_css`` block along with any other page-specific bundles:"

Author: Good idea!

.. code-block:: html

   {% block page_css %}
        {{ css_bundle('protocol-call-out') }}

>     {% endblock %}

Reviewer: Indentation

.. code-block:: html

   {% block page_css %}
        {{ css_bundle('protocol-split') }}

>     {% endblock %}

Reviewer: Indentation
